pkgs/by-name/db/dbeaver-bin/package.nix

```diff
@@ -12,6 +12,7 @@
   glib,
   webkitgtk_4_0,
   glib-networking,
+  settings ? { Xmx = "1024m"; },
```

Comment:
Given that this supports only xmx we should not make this an attribute and just have a key like override_xmx

Reply:
Fine by me. :+1: 
For context: I was going to allow overriding more but didn't in the end.